dyld[5478]: Library not loaded: @loader_path/../Python
  Referenced from: <DAED6A8B-509E-3042-B1F9-2F792191741A> /Users/vladimir/Library/Application Support/pypoetry/venv/bin/python3.11
  Reason: tried: '/Users/vladimir/Library/Application Support/pypoetry/venv/bin/../Python' (no such file), '/usr/local/lib/Python' (no such file), '/usr/lib/Python' (no such file, not in dyld cache)

Traceback:

  File "<stdin>", line 923, in main
  File "<stdin>", line 562, in run
